docker-compose/is/conf/identity-server/repository/conf/deployment.toml

```diff
@@ -13,7 +13,9 @@ type = "read_write_ldap_unique_id"
 connection_url = "ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}"
 connection_name = "uid=admin,ou=system"
 connection_password = "admin"
-base_dn = "dc=wso2,dc=org"      #refers the base dn on which the user and group search bases will be generated
+base_dn = "dc=wso2,dc=org"
+#refers the base dn on which the user and group search bases will be generated
```

Comment:
I thought the typical convention was that if a comment wasn't on the same line then it was above the related code.